Firestore/Source/Local/FSTLRUGarbageCollector.h

```diff
@@ -107,4 +159,6 @@ extern const firebase::firestore::model::ListenSequenceNumber kFSTListenSequence
 
 - (size_t)byteSize;
 
+- (FSTLruGcResults)tryRunGcWithLiveTargets:(NSDictionary<NSNumber *, FSTQueryData *> *)liveTargets;
```

Comment:
Instead of "run GC" as the verb here, maybe just "collect"? Also, can we just define collection to be a thing that's attempted rather than definite? If so this could be `collectWithLiveTargets` which would be cleaner (and just Collect once this is ported to C++).